web/packages/teleterm/src/ui/services/workspacesService/documentsService/types.ts

```diff
@@ -167,6 +167,11 @@ export interface DocumentConnectMyComputer extends DocumentBase {
   // However, there are a few components in the system, such as `getResourceUri`, which need to determine the relation
   // between a document and a cluster just by looking at the document fields.
   rootClusterUri: uri.RootClusterUri;
+  /**
+   * The status of 'connecting' is used to indicate that Connect My Computer permissions cannot be
+   * established yet and the document is waiting for the app to receive full cluster details.
+   */
+  status: '' | 'connecting' | 'connected' | 'error';
```

Comment:
This is kind of weird since technically we're not connecting anywhere. But that's the values that existing tabs already use to trigger the loading state on the tab, so I just reused it here.

I cannot just change them because this state gets persisted to disk and I'd first need to check if changing the values in other tabs to something different won't break people who already have tabs with `connected` etc. persisted to disk.

I also didn't want to add a second set of values to avoid a situation where some tabs use the existing values and some use, say, `processing, success, error`.